### ISSUE TITLE
Add option to reverse behaviour and not block ads by default

### ISSUE THREAD

**Issue by jglass071** [uBlockOrigin/uBlock-issues#98]
### Prerequisites

- [x] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.

### Description

Websites on the Internet rely on advertising revenue. Some users may prefer to support all the websites they visit by leaving ads unblocked automatically, and manually blocking websites where the ads are intrusive on a site-by-site basis. The extension would in effect work in reverse to what it does now if this option is enabled.

**Comment by uBlock-user** [comment]
Duplicate of https://github.com/gorhill/uBlock/issues/1698

**Comment by gwarser** [comment]
I added section about this here https://github.com/gorhill/uBlock/wiki/How-to-whitelist-a-web-site to not link directly to dynamic filtering sub-pages - we need to inform about dynamic filtering and about "advanced users" mode.
